tests/expr_and_series/abs_test.py

```diff
@@ -1,14 +1,10 @@
 from typing import Any
 
-import pytest
-
 import narwhals.stable.v1 as nw
 from tests.utils import compare_dicts
 
 
-def test_abs(constructor: Any, request: Any) -> None:
-    if "dask" in str(constructor):
-        request.applymarker(pytest.mark.xfail)
+def test_abs(constructor: Any) -> None:
```

Comment:
If you roll back these changes, as they are addressed in the other PR, all CI should get green and we will be able to merge

Reply:
yup - or alternatively, fetch upstream and rebase against `upstream/main`